docs/ibm-cloud-rules.md

```diff
@@ -56,10 +56,10 @@ which is delivered in the `@ibm-cloud/openapi-ruleset` NPM package.
   * [ibm-inline-response-schema](#ibm-inline-response-schema)
   * [ibm-major-version-in-path](#ibm-major-version-in-path)
   * [ibm-merge-patch-properties](#ibm-merge-patch-properties)
-  * [ibm-operation-id-case-convention](#ibm-operation-id-case-convention)
-  * [ibm-operation-id-naming-convention](#ibm-operation-id-naming-convention)
-  * [ibm-operation-summary](#ibm-operation-summary)
-  * [ibm-optional-request-body](#ibm-optional-request-body)
+  * [ibm-operation-summary-exists](#ibm-operation-summary-exists)
+  * [ibm-operationid-casing-convention](#ibm-operationid-casing-convention)
+  * [ibm-operationid-naming-convention](#ibm-operationid-naming-convention)
+  * [ibm-optional-requestbody](#ibm-optional-requestbody)
```

Comment:
Was this change made because `requestBody` is the actual OpenAPI field? Just curious

Reply:
Yes, I think we were supposed to use the actual fieldname but convert to lower-case since rule ids must be lower kebab case.  Does that sound right? (same thing I did with operationId -> operationid in a couple other rule ids

Reply:
Yep, I noticed it was consistent with operationid. I think that sounds good to me 👍